transport/src/main/java/io/netty/channel/ChannelConfig.java

```diff
@@ -195,17 +195,13 @@
     ChannelConfig setAutoRead(boolean autoRead);
 
     /**
-     * @deprecated From version 5.0, {@link Channel} will not be closed on write failure.
```

Comment:
@buchgr we should still keep the `@deprecated`javadoc (just remove the reference to 5.0).